README.md

```diff
@@ -50,6 +50,11 @@ const matic = new Matic({
 // Warning: Not-safe
 // matic.wallet = <private-key> // Use metamask provider or use WalletConnect provider instead.
 
+// get token address mapped with mainchain token address
+await matic.getMappedTokenAddress(
```

Comment:
```suggestion
const tokenAddressOnMatic = await matic.getMappedTokenAddress(
```

Reply:
fixed in: https://github.com/maticnetwork/matic.js/pull/18